From 6ced29e6d7ba6ff657c8431fda6039267a3eb39f Mon Sep 17 00:00:00 2001
From: Jacob Champion <jacob.champion@ni.com>
Date: Fri, 22 Nov 2013 10:47:20 -0600
Subject: [PATCH 13/14] crypto: Use i686 instead of i486 in MASM script

Although technically MASM is unsupported, we can't use NASM in Build
Services (or at least not at the moment). Luckily, Windows 32- and
64-bit builds for 1.0.1e appear to work fine with MASM -- except for the
x86*cpuid assembly files, which use instructions that weren't introduced
until Intel 686. This is acceptable for us on Windows, so bump the ISA
in the MASM generator script.

Natinst-ReviewBoard-ID: 53579

Upstream-Status: Inappropriate [NIBuild support]
---
 crypto/perlasm/x86masm.pl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/crypto/perlasm/x86masm.pl b/crypto/perlasm/x86masm.pl
index 1741342..786b3c4 100644
--- a/crypto/perlasm/x86masm.pl
+++ b/crypto/perlasm/x86masm.pl
@@ -82,7 +82,7 @@ TITLE	$_[0].asm
 IF \@Version LT 800
 ECHO MASM version 8.00 or later is strongly recommended.
 ENDIF
-.486
+.686
 .MODEL	FLAT
 OPTION	DOTNAME
 IF \@Version LT 800
-- 
2.6.2

